packages/react-interpolate-components/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "react-interpolate-components",
+  "version": "0.0.0",
+  "description": "Convert strings into structured React components.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Automattic/wp-calypso/tree/master/packages/react-interpolate-components"
+  },
+  "main": "lib/index.js",
+  "keywords": [
+    "react",
+    "react-component",
+    "interpolation"
+  ],
+  "engines": {
+    "node": ">=4.2.3"
+  },
+  "scripts": {
+	"compile": "babel -sd lib/ src/",
```

Comment:
Doesn't bother me much, but worth considering: Should we have scripts split like we do here? i.e. `make test` and `npm run compile`, when `test` could be an npm script or `compile` a Make target.